src/diagrams/gantt/ganttDb.js

```diff
@@ -58,7 +63,22 @@ export const getTasks = function () {
   return tasks
 }
 
-const getStartDate = function (prevTime, dateFormat, str) {
+const isInvalidDate = function (date, dateFormat, excludes) {
+  if (date.isoWeekday() >= 6 && excludes.indexOf('weekends') >= 0) {
```

Comment:
Everywhere it says weekdays, (even in the PR title) but the code only checks for the word 'weekends'. The latter seems more logical, because people are more likely to wanting to exclude weekends. Although that has problems attached to it as well, which I will address in the general review comment.

Reply:
Correct. I kept weekend as the "default" but if the user chooses to put the weekday name (friday, saturday, monday), it will work too.